app/frontend/utils/export-permit-application-csv.ts

```diff
@@ -0,0 +1,204 @@
+import { format } from "date-fns"
+import * as R from "ramda"
+import { IPermitApplication } from "../models/permit-application"
+import { ERequirementType } from "../types/enums"
+import { startBlobDownload, urlForPath } from "./utility-functions"
+
+type TRow = {
+  section: string
+  panel: string
+  multi: string
+  label: string
+  value: string
+}
+
+const readValue = (submissionData: any, path: (string | number)[]) => R.path(path, submissionData)
+
+const optionValueToLabel = (component: any, value: any) => {
+  if (!component?.values || value == null) return value
+  const found = (component.values || []).find((opt: any) => opt.value === value)
+  return found?.label ?? value
+}
+
+const resolveComponentDisplay = (component: any, value: any): string => {
+  const type = component?.type
+  switch (type) {
+    case ERequirementType.radio:
+    case ERequirementType.select:
+    case ERequirementType.multiOptionSelect:
+      return String(optionValueToLabel(component, value) ?? "")
+    case "simplefile":
+    case ERequirementType.file: {
+      const files = Array.isArray(value) ? value : []
+      if (files.length === 0) return ""
+
+      const fileSummaries = files.map((f: any) => {
+        const original = f?.originalName || f?.name
+        const apiUrl =
+          f?.model && f?.modelId ? urlForPath(`/api/s3/params/download?model=${f.model}&modelId=${f.modelId}`) : null
+        const path = f?.fileUrl || f?.url || apiUrl || f?.filename || f?.id
+        if (original && path) return `${original} → ${path}`
+        if (path) return String(path)
+        return original ? String(original) : "(file)"
+      })
+      return `(file uploaded: ${fileSummaries.join("; ")})`
+    }
+    case ERequirementType.checkbox: {
+      return value ? "Yes" : value === false ? "No" : ""
+    }
+    case "checklist": {
+      if (!value || typeof value !== "object") return ""
+      const selected = Object.keys(value).filter((k) => !!value[k])
+      if (!selected.length) return ""
+
+      const labelMap = new Map<string, string>()
+      ;(component.values || []).forEach((opt: any) => labelMap.set(opt.value, opt.label))
+      return selected.map((v) => labelMap.get(v) || v).join("; ")
+    }
+    case ERequirementType.signature:
+      return value ? "(signed)" : ""
+    case ERequirementType.text:
+    case ERequirementType.number:
+    case ERequirementType.date:
+    case ERequirementType.address:
```

Comment:
Can you confirm that we don't need:
ERequirementType.select
ERequirementType.multiOptionSelect
radio, multiplaysumgrid, etc.